test/data/register-a-birth-questions-and-responses.yml

```diff
@@ -10,6 +10,7 @@
 - papua-new-guinea # registration_can_take_3_months
 - philippines # oru_extra_documents_in_philippines_when_mother_not_british
 - sierra-leone # oru_extra_documents_variant_intro
+- somalia
```

Comment:
Reading the commit note, I wasn't sure how the artefacts had been created i.e. it wasn't obvious that you had generated them by adding `somalia` to this question in the questions and responses YAML file. This was exacerbated, because the diff of the latter file appears below all the artefact diffs. Although it seemed unlikely, it crossed my mind that you might've generated them by hand somehow. I think it would be worth explaining a little more about how you generated the artefacts in this commit.